requirements/dev.in

```diff
@@ -0,0 +1,16 @@
+-c constraints.txt
+-r test.txt
+
+# Debug tools
+django-debug-panel
```

Comment:
As noted in https://github.com/edx/upgrades/issues/77 , this dependency shouldn't be needed anymore.